SteakBot.Core/Modules/MemeModule.cs

```diff
@@ -20,20 +20,50 @@ public MemeModule(MemeService memeService)
         public async Task AddMeme(string name, string value, string description)
         {
             var replyMessage = string.Empty;
-            EmbedBuilder embed = null;
+            EmbedBuilder embed = new EmbedBuilder();
```

Comment:
Both local fields are now unused and obsolete.

Reply:
Corrected.